app/services/integrations/aggregator/taxes/invoices/create_service.rb

```diff
@@ -45,7 +45,9 @@ def payload
 
             invoice_data = payload_body.first
             invoice_data["id"] = invoice.id
-            invoice_data["type"] = "salesInvoice" if integration.type.to_s == "Integrations::AvalaraIntegration"
+            if integration.type.to_s == "Integrations::AvalaraIntegration"
+              invoice_data["type"] = invoice.voided? ? "returnInvoice" : "salesInvoice"
```

Comment:
this branching is not covered in the service test

Reply:
This is private method so I haven't tested it directly, but in the spec I am providing mocked response based on expected payload (lines 156 and 226 in spec) and there are two different expected payloads, one for each case